### ISSUE TITLE
www.aetv.com  help

### ISSUE THREAD

**Issue by zsquared24** [reek/anti-adblock-killer#1994]
Was working great now they are bypassing and I get the message again to disable adblock have to watch commercials again just ads wouldnt be bad this sucks please how can I fix zsquared24@gmail.com

**Comment by zsquared24** [comment]
I added filters that you gave the lady for mylifetime.com they worked thanks so much this fixed it THANKS USER MAPX all credit to u
@@||aetv.com^$generichide
@@*$image,domain=aetv.com
@@||budgetedbauer.com$domain=aetv.com

**Comment by OverlordQ** [comment]
Doesn't fix it for me

**Comment by zsquared24** [comment]
It no longer works for me either OverlordQ I am currently seeking new filters to use.

**Comment by OverlordQ** [comment]
Traditional
```
@@||aetv.com^$generichide
@@*$image,domain=aetv.com
@@||budgetedbauer.com$domain=aetv.com
```
Doesn't seem to be working on aetv anymore.

**Comment by mapx-** [comment]
try
`@@||fastly.net/ad/$domain=aetv.com`

or provide free show / episode to test without any provider..

**Comment by OverlordQ** [comment]
Yeah fell asleep before mentioned found that on another thread and it seemed to work.

**Comment by zsquared24** [comment]
I updated my adblock and it worked havent been the site in a while though

**Comment by OverlordQ** [comment]
Worked yesterday, but throwing error again today with fastily rule.

**Comment by OverlordQ** [comment]
http://www.aetv.com/shows/intervention/season-1/episode-1 should be available to test with w/o provider logins

**Comment by mapx-** [comment]
the video is starting fine, no anti adblocking issue, use only 1 adblocker, no other ghostery, disconnect, anti banner ..

**Comment by OverlordQ** [comment]
Weird, it's working again, after like the sixth time I purged cache. Any additional debugging I can try from my end next time I run into it?
